### DISCUSSION TITLE
add Manifest for hub.docker.com

### DISCUSSION THREAD

**Discussion by erikderzweite** [lukevella/rallly#157] [General]
The image from hub.docker.com does not seem to have a manifest, is that by design?

```
docker pull lukevella/rallly
Using default tag: latest
Error response from daemon: manifest for lukevella/rallly:latest not found: manifest unknown: manifest unknown
```

**Comment by lukevella** [comment]
I haven't published an image to docker hub yet but I will. It will happen as part of a bigger update.

**Reply by samsapti** [reply]
Hey @lukevella, when do you expect to publish a Docker image?

**Reply by lukevella** [reply]
@samsaptidev It's not a priority at the moment. Unfortunately it's not as trivial as it may seem since next.js projects requires specifying environment variables during build time. I'm sure there's some solution for this but as it stands I don't see it being worthwhile to invest the time to figure this out since cloning the repo and building an image should work just fine. I'd need to take time away from delivering more features and making UI improvements to make this happen and I haven't heard a strong case for that.

**Reply by samsapti** [reply]
@lukevella Okay I see, though I can see several reasons for why a prebuilt image would be favorable:

- When building the image yourself, it's not possible to have it automatically update via tools like Watchtower.
- Building the image might be too heavy on low-end hardware.
- I don't think it's too trivial to use it in an Ansible setup when you have to build the image yourself.

If you want me to, I can help take a look at it. I have a good deal of Docker experience.
